src/biogeochem/DustEmisLeung2023.F90

```diff
@@ -0,0 +1,874 @@
+module DustEmisLeung2023
```

Comment:
Just casually asking, but it might be better to take away 2023 and just use DustEmisLeung, because of the potential upcoming small updates of my scheme... For DustEmisZender2003 it's good to keep 2003 though because Zender will probably update his scheme. Just curious here on what is a better way to do it.

Reply:
Hmmm. Good question. The 2023 gets you the paper to look for. I think small tuning and adjustments that don't need a new paper can continue to be under that. But, anything that justifies a new paper -- can have the new year for the updated paper (so DustEmisLeung2025 for example). If you remove the year there isn't a way to distinguish between different versions of Leung2023. For CESM3.0 we need a single version that we identify, but after CESM3.0 you might make further adjustments and that could have a new year to signal it's got important differences in answers from CESM3.0. So including the year makes sense for that kind of thing.

If we think about all the options that we could have at some future date it might be a list like:

- NoDustEmis
- DustEmisZender2003 (clm5.0 and before)
- DustEmisKok2014
- DustEmisLeung2023 (clm6.0)
- DustEmisLeung2026 (clm6.5 or clm7.0?)
- DustEmisPrescribed2027

There wouldn't be a version at a frequency less than every few years, so the year is sufficient. Changes less than that frequency might be considered a bug fix, rather than a new version. But, this also allows each author to have multiple versions to their name and series. So if Kok or Zender took it up again there could be multiple versions of their method.

So I think the year is important to keep in.

Reply:
Yea, your thoughts all make sense to me. So be it. Thanks!